src/cli/traceViewer/traceViewer.ts

```diff
@@ -140,8 +140,12 @@ export async function showTraceViewer(traceStorageDir: string | undefined, trace
   let files: string[];
   if (fs.statSync(tracePath).isFile()) {
     files = [tracePath];
-    if (!traceStorageDir)
+    if (!traceStorageDir) {
       traceStorageDir = path.dirname(tracePath);
+
+      if (fs.existsSync(traceStorageDir + '/trace-resources'))
```

Comment:
Let's move this right before the line 155, to cover the "tracePath is a directory" case as well?

Reply:
Wouldn't that potentially break for the case when user runs `npx playwright show-trace --resource /project/trace-resources /project/trace.ndjson` ? I guess that would only break if `/project/trace-resources/trace-resources` would also exist...

Reply:
Good point. Let's copy this heuristic to line 152 then?